macosx/HBTitle.m

```diff
@@ -278,4 +306,9 @@ - (NSDictionary *)jobSettingsWithPreset:(HBPreset *)preset
     return result;
 }
 
+- (HBMetadata *)metadata
```

Comment:
This is not needed, the compiler should synthesise this code automatically.

Reply:
- [ ] remove this statement